### ISSUE TITLE
light mode

### ISSUE THREAD

**Issue by teambitcoin** [pox/EndTheFUD#27]
should enable a light mode toggle for the website and Review Accessibility ranking. This will impact search engine visibility. The site should be visually impaired friendly.

**Comment by teambitcoin** [comment]
*** if its not already.

**Comment by pox** [comment]
If anyone has an idea how to do this with GitHub Pages & Jekyll themes I'd love to know.
